src/main/scala/no/ndla/searchapi/service/search/MultiDraftSearchService.scala

```diff
@@ -77,7 +79,21 @@ trait MultiDraftSearchService {
           )
       })
 
-      val boolQueries: List[BoolQuery] = List(contentSearch, noteSearch).flatten
+      val embedResourceSearch = settings.embedResource.map(q => {
+        boolQuery()
+          .should(
+            simpleStringQuery(q).field("embedResources", 1),
+          )
+      })
+
+      val embedIdSearch = settings.embedId.map(q => {
+        boolQuery()
+          .should(
+            simpleStringQuery(q).field("embedIds", 1),
+          )
+      })
```

Comment:
Jeg syns det ser ut som disse kanskje burde vært filtere istedet for `simpleStringQuery`'er. Kan det stemme?